bedrock/firefox/templates/firefox/flashback/index.de.html

```diff
@@ -0,0 +1,38 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_image %}{{ static('img/firefox/flashback/og-de.jpg') }}{% endblock %}
+{% block page_favicon %}{{ static('img/favicons/firefox/browser/favicon.ico') }}{% endblock %}
+{% block page_favicon_large %}{{ static('img/favicons/firefox/browser/favicon-196x196.png') }}{% endblock %}
+{% block page_ios_icon %}{{ static('img/favicons/firefox/browser/apple-touch-icon.png') }}{% endblock %}
+
+{# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
+{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
+
+{% block page_title %}2019 Firefox Flashback{% endblock %}
+
+{% block page_desc %}Ein Jahr im Web: Finde heraus, was du 2019 online so erlebt hast.{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('flashback') }}
+{% endblock %}
+
+{% block body_id %}firefox-flackback{% endblock %}
+{% block body_class %}firefox-flackback mzp-t-firefox{% endblock %}
+
+{% block content %}
+  <main role="main">
+    <div class="c-panel t-welcome-de">
+      <div class="mzp-l-content">
+        {{ high_res_img('protocol/img/logos/firefox/browser/logo-lg.png', {'alt': 'Firefox', 'width': '100', 'height': '100', 'class': 'c-welcome-logo'}) }}
+        <h1>Mööp! Wir haben keinen blassen Schimmer, was du dieses Jahr online gemacht hast. </h1>
+        <p>Interessiert uns auch nicht.</p>
+        <p>Wir stehen für Privatsphäre. Punkt.</p>
+        <div><a class="mzp-c-button mzp-t-product" data-cta-type="link" data-cta-text="Wieso Weshalb Warum" href="https://blog.mozilla.org/firefox/de/firefox-versprechen-fuer-deine-persoenlichen-daten/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=flashback">Wieso? Weshalb? Warum?</a></div>
```

Comment:
/firefox/privacy/ looks like it's available in German, if we'd prefer to link there instead of the blog post

Reply:
Nope, they want the blog post 🤷‍♀